types.go

```diff
@@ -175,6 +183,10 @@ type (
 		// Defaults to 'Single'
 		AllowedFunctionsPerContainer AllowedFunctionsPerContainer `json:"allowedFunctionsPerContainer"`
 	}
+	EnvVar struct {
```

Comment:
Hmmm, let's not do this.  I get why you're trying to avoid pulling in the whole k8s API package in here, but I think that's better than duplicating little pieces of it in incomplete ways. For example this is missing `EnvVarSource`, which is in fact the right way to solve the problem you're trying to solve (because it handles secrets correctly).  So we should just use an actual K8s EnvVar slice and copy it into pod spec's container spec.

Reply:
Yeah I am also okay with that. 👍

Reply:
👍 thanks!